### DISCUSSION TITLE
Validation of the form fields

### DISCUSSION THREAD

**Discussion by SashaDorfman** [cloudscape-design/components#798] [Q&A] [answered]
Hello ! I'm putting together a form using Cloudscape. How do I validate input fields ? 
For example, I went through [`Input` component API ](https://cloudscape.design/components/input/?tabId=api ), but I didn't see anything about validation. 
Thank you !

**Comment by just-boris** [comment, marked as answer]
Hello!

Validation messages should be provided via FormField component

```js
<FormField errorText={validateValue(value)}>
  <Input value={value}>
</FormField>
```

Check this codesanbox for a full example: https://codesandbox.io/s/cloudscape-min-max-05xtb4?file=/src/App.tsx
